bi-connectors/PowerBIConnector/docs/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to the connector will be documented in this file.
```

Comment:
I'd explicitly say OpenSearch Power BI Connector instead of "the connector"

Reply:
In this case should we have two changelogs? One for `OpenSearch Project` and one for `Amazon OpenSearch Service`